packages/calcite-components/src/components/button/button.scss

```diff
@@ -228,271 +253,288 @@
 :host([appearance]) {
   button,
   a {
-    @apply border-color-transparent
-      border
-      border-solid;
+    background-color: var(--calcite-button-background-color);
+    color: var(--calcite-button-text-color);
+    border: var(--calcite-border-width-sm) solid var(--calcite-button-border-color);
+    box-shadow: var(--calcite-button-box-shadow);
   }
 }
 
 :host([kind="brand"]) {
   button,
   a {
-    @apply text-color-inverse bg-brand;
+    --calcite-button-text-color: var(--calcite-color-text-inverse);
+    --calcite-button-background-color: var(--calcite-color-brand);
```

Comment:
Looks like you forgot to assign `color` and `background-color` here.  Applies to many other places below too.  See my earlier comment about revising this to assign all the variants of these css properties at the top of the file.